content/4_Gold/Sliding_Window.problems.json

```diff
@@ -190,6 +190,18 @@
     }
   ],
   "queue": [
+    {
+      "uniqueId": "cses-3221",
+      "name": "Sliding Window Minimum",
+      "url": "https://cses.fi/boi24/task/3221/",
+      "source": "CSES",
+      "difficulty": "Easy",
+      "isStarred": true,
+      "tags": [],
+      "solutionMetadata": {
+        "kind": "internal"
+      }
+    },
```

Comment:
how's this different from the sliding window maximum problem

Reply:
Nothing, it's just minimum

Reply:
uh
not really sure why we should add this then

Reply:
Just think like we not only solve "nearest greater element" prblm but also the "nearest smallest element".

Reply:
i mean idk why that problem was added
sorry this just seems, really redundant

Reply:
Okay if you say so

Reply:
> Nothing, it's just minimum

so it's just changing min to max? the module has sliding window code already

Reply:
yes i will going to remove it